EAD-Assignment-5/src/com/metacube/advertismentwebservices/resources/AdvertismentResource.java

```diff
@@ -0,0 +1,138 @@
+package com.metacube.advertismentwebservices.resources;
+
+import java.util.List;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.HeaderParam;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import com.google.gson.Gson;
+import com.metacube.advertismentwebservices.model.Advertisement;
+import com.metacube.advertismentwebservices.Common.Status;
+import com.metacube.advertismentwebservices.facade.AdvertisementFacade;
+
+@Path("/Advertisement")
+public class AdvertismentResource {
+	AdvertisementFacade advertisementFacade = AdvertisementFacade.getInstance();
+
+	/**
+	 * Function to insert data in database table
+	 * 
+	 * @param id
+	 * @param name
+	 * @return status
+	 */
+	@POST
+	@Path("/InsertAdvertisement")
+	@Produces(MediaType.APPLICATION_JSON)
+	@Consumes(MediaType.APPLICATION_JSON)
+	public Status insertIntoAdvertisement(
+			@HeaderParam("Authorization") String authorizationString,
+			Advertisement advertisement) {
+
+		if (!"GET-2018".equals(authorizationString)) {
```

Comment:
This method seems repetitive, can create a separate method to check authorization